package.json

```diff
@@ -50,8 +50,7 @@
   "scripts": {
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "start": "react-scripts start",
-    "test": "react-scripts test"
+    "start": "react-scripts start"
```

Comment:
Note that I removed the test target from the here. There are no tests and the execution fails when the typescript workflow runs them in CI. If a target is specified, it must be expected to work.